aws/resource_aws_eip_test.go

```diff
@@ -1171,3 +1199,12 @@ resource "aws_eip" "test" {
 }
 `
 }
+
+const testAccAWSEIPConfigNetworkBorderGroup = `
+data "aws_region" current {}
+
+resource "aws_eip" "test" {
+  vpc  				   = "true"
+  network_border_group = data.aws_region.current.name
```

Comment:
Since this pull request was introduced newer CI testing would fail here after merge, so fixing it ahead of time:

```suggestion
  vpc                  = true
  network_border_group = data.aws_region.current.name
```